dataprep/data_connector/implicit_database.py

```diff
@@ -67,8 +67,9 @@ def __init__(self, name: str, config: Dict[str, Any]) -> None:
         )  # This will throw errors if validate failed
         self.name = name
         self.config = config
-
         request_def = config["request"]
+        if config["request"]["paganition"]["type"] == "limit":
```

Comment:
I didn't see any usage of these two lines

Reply:
removed.